egs2/clarity/enh_2021/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+sample_rate=44k
+# Path to a directory containing extra annotations for CHiME4
+# Run `local/data.sh` for more information.
+extra_annotations=
+
+
+
+train_set=train
+valid_set=dev
+test_sets="dev"
+
+./enh.sh \
+    --train_set "${train_set}" \
+    --valid_set "${valid_set}" \
+    --test_sets "${test_sets}" \
+    --fs ${sample_rate} \
+    --ngpu 1 \
+    --spk_num 1 \
+    --ref_channel 0 \
+    --local_data_opts "--extra-annotations ${extra_annotations} --stage 1 --stop-stage 2" \
+    --enh_config conf/tuning/train_enh_beamformer_mvdr.yaml \
+    --use_dereverb_ref false \
+    --use_noise_ref true \
```

Comment:
Maybe it's better to set this to `false` if no parallel noise data is provided for training.